### ISSUE TITLE
Bug on Folia - Multiple Skill

### ISSUE THREAD

**Issue by Euphillya** [mcMMO-Dev/mcMMO#5092]
spam console : 

```
[20:09:10 WARN]: [mcMMO] Entity task for mcMMO v2.2.018 generated an exception
java.lang.IllegalStateException: Accessing entity state off owning region's thread
        at ca.spottedleaf.moonrise.common.util.TickThread.ensureTickThread(TickThread.java:59) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.entity.CraftPigZombie.getHandle(CraftPigZombie.java:42) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.entity.CraftPigZombie.getHandle(CraftPigZombie.java:7) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.entity.CraftEntity.isInWorld(CraftEntity.java:947) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.entity.CraftEntity.isValid(CraftEntity.java:401) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at mcMMO.jar/com.gmail.nossr50.runnables.skills.RuptureTask.run(RuptureTask.java:45) ~[mcMMO.jar:?]
        at mcMMO.jar/com.gmail.nossr50.util.CancellableRunnable.accept(CancellableRunnable.java:20) ~[mcMMO.jar:?]
        at mcMMO.jar/com.gmail.nossr50.util.CancellableRunnable.accept(CancellableRunnable.java:7) ~[mcMMO.jar:?]
        at mcMMO.jar/com.gmail.nossr50.mcmmo.folialib.impl.FoliaImplementation.lambda$runAtEntityTimer$21(FoliaImplementation.java:342) ~[mcMMO.jar:?]
        at io.papermc.paper.threadedregions.scheduler.FoliaEntityScheduler$EntityScheduledTask.accept(FoliaEntityScheduler.java:170) ~[bloomexcalia-1.21.1.jar:?]
        at io.papermc.paper.threadedregions.scheduler.FoliaEntityScheduler$EntityScheduledTask.accept(FoliaEntityScheduler.java:117) ~[bloomexcalia-1.21.1.jar:?]
        at io.papermc.paper.threadedregions.EntityScheduler.executeTick(EntityScheduler.java:181) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1711) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at io.papermc.paper.threadedregions.TickRegions$ConcreteRegionTickHandle.tickRegion(TickRegions.java:407) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at io.papermc.paper.threadedregions.TickRegionScheduler$RegionScheduleHandle.runTick(TickRegionScheduler.java:418) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at ca.spottedleaf.concurrentutil.scheduler.SchedulerThreadPool$TickThreadRunner.run(SchedulerThreadPool.java:540) ~[folia-1.21.1.jar:1.21.1-26-b440e32]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
        ```

**Comment by Euphillya** [comment]
```yml
[18:38:31 WARN]: [mcMMO] Entity task for mcMMO v2.2.018 generated an exception
java.lang.IllegalStateException: Accessing entity state off owning region's thread
        at ca.spottedleaf.moonrise.common.util.TickThread.ensureTickThread(TickThread.java:59) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.entity.CraftZombie.getHandle(CraftZombie.java:24) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.entity.CraftZombie.getHandle(CraftZombie.java:9) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.entity.CraftLivingEntity.getHealth(CraftLivingEntity.java:100) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at mcMMO.jar/com.gmail.nossr50.runnables.skills.AwardCombatXpTask.run(AwardCombatXpTask.java:30) ~[mcMMO.jar:?]
        at mcMMO.jar/com.gmail.nossr50.util.CancellableRunnable.accept(CancellableRunnable.java:20) ~[mcMMO.jar:?]
        at mcMMO.jar/com.gmail.nossr50.util.CancellableRunnable.accept(CancellableRunnable.java:7) ~[mcMMO.jar:?]
        at mcMMO.jar/com.gmail.nossr50.mcmmo.folialib.impl.FoliaImplementation.lambda$runAtEntity$15(FoliaImplementation.java:259) ~[mcMMO.jar:?]
        at io.papermc.paper.threadedregions.scheduler.FoliaEntityScheduler$EntityScheduledTask.accept(FoliaEntityScheduler.java:170) ~[bloomexcalia-1.21.1.jar:?]
        at io.papermc.paper.threadedregions.scheduler.FoliaEntityScheduler$EntityScheduledTask.accept(FoliaEntityScheduler.java:117) ~[bloomexcalia-1.21.1.jar:?]
        at io.papermc.paper.threadedregions.EntityScheduler.executeTick(EntityScheduler.java:181) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1711) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at io.papermc.paper.threadedregions.TickRegions$ConcreteRegionTickHandle.tickRegion(TickRegions.java:407) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at io.papermc.paper.threadedregions.TickRegionScheduler$RegionScheduleHandle.runTick(TickRegionScheduler.java:418) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at ca.spottedleaf.concurrentutil.scheduler.SchedulerThreadPool$TickThreadRunner.run(SchedulerThreadPool.java:540) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
        ```

**Comment by Euphillya** [comment]
New error :

```
[22:29:22 ERROR]: Could not pass event PlayerInteractEvent to mcMMO v2.2.018
java.lang.IllegalArgumentException: amount must be greater than 0
        at com.google.common.base.Preconditions.checkArgument(Preconditions.java:143) ~[guava-32.1.2-jre.jar:?]
        at org.bukkit.inventory.ItemStack.of(ItemStack.java:59) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:138) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:117) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:104) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at mcMMO.jar/com.gmail.nossr50.util.ChimaeraWing.getChimaeraWing(ChimaeraWing.java:151) ~[mcMMO.jar:?]
        at mcMMO.jar/com.gmail.nossr50.util.ChimaeraWing.activationCheck(ChimaeraWing.java:99) ~[mcMMO.jar:?]
        at mcMMO.jar/com.gmail.nossr50.listeners.PlayerListener.onPlayerInteractMonitor(PlayerListener.java:882) ~[mcMMO.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor449.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:1.21.1-26-b440e32]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:604) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:560) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:555) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:551) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItem(ServerGamePacketListenerImpl.java:2057) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:47) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:10) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:58) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$2(PacketUtils.java:86) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at io.papermc.paper.threadedregions.EntityScheduler.executeTick(EntityScheduler.java:181) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1711) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at io.papermc.paper.threadedregions.TickRegions$ConcreteRegionTickHandle.tickRegion(TickRegions.java:407) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at io.papermc.paper.threadedregions.TickRegionScheduler$RegionScheduleHandle.runTick(TickRegionScheduler.java:418) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at ca.spottedleaf.concurrentutil.scheduler.SchedulerThreadPool$TickThreadRunner.run(SchedulerThreadPool.java:540) ~[bloomexcalia-1.21.1.jar:1.21.1-26-b440e32]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

**Comment by nossr50** [comment]
Fixed in 2.2.019
